src/main/tables/90_quick_launches.sql

```diff
@@ -2,16 +2,22 @@ SET search_path = public, pg_catalog;
 
 -- Associates information needed to manage badges.
 
-CREATE TABLE badges (
+CREATE TABLE quick_launches (
   -- Primary key
   id uuid NOT NULL,
 
   -- The name of the badge. Same length as the name in the apps table.
   name character varying(255) NOT NULL,
 
+  -- A description of the quick launch.
+  description text,
+
   -- Foreign key into the users table
-  user_id uuid NOT NULL,
+  creator uuid NOT NULL,
 
   -- Foreign key into the submissions table
-  submission_id uuid NOT NULL
+  submission_id uuid NOT NULL,
+
+  -- Whether or not all users can see the quick launch
+  public boolean NOT NULL DEFAULT false
 );
```

Comment:
It seems to me as though it might be useful to have a quick way to associate a quick launch with the app that it uses. Unless I'm missing something, it appears that the only way to make that association in all cases at this time is to examine the submission itself, which might be a little slower than we'd like unless we add an index. It does appear that we can add an index on a JSON field, so that is a viable option. The option that the `jobs` table uses is to store the app ID directly in the table as well. I think that either option would work although having some information duplicated in the table might be less than ideal. It might be necessary to experiment to see which option is best.

Reply:
Good catch, I completely forgot to add the app_id column. I meant to, just missed it.

Reply:
This should be fixed now.